src/Microsoft.TemplateEngine.Authoring.TemplateVerifier/Commands/Assertions/CommandResultAssertions.cs

```diff
@@ -0,0 +1,186 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#if NET
+
+using System.Text.RegularExpressions;
```

Comment:
I haven't reviewed those files. I think it would be good to extract them to separate package instead.

Reply:
They are not needed here - removing altogether